main.nf

```diff
@@ -1210,7 +1210,7 @@ process samtools_flagstat {
     file(bam) from ch_mapping_for_samtools_flagstat
 
     output:
-    file "*.stats" into ch_flagstat_for_multiqc
+    tuple val(prefix), file("*stats") into ch_flagstat_for_multiqc,ch_flagstat_for_endorspy
```

Comment:
Nice that you use the tuple !
It begs for another PR to replace all the `set` (being deprecated) by `tuple` [https://www.nextflow.io/docs/latest/process.html?highlight=tuple#input-of-type-tuple](https://www.nextflow.io/docs/latest/process.html?highlight=tuple#input-of-type-tuple)

Reply:
I think that is the plan with the TSV input (as we have to update all the input/outputs anyway)